server/views/partials/applicationInsightsScript.njk

```diff
@@ -0,0 +1,209 @@
+<script nonce="{{ cspNonce }}" type="text/javascript">
```

Comment:
Ah this addresses my comment from an earlier commit - looks like a good approach to me.

Reply:
Actually I wanted to check with you if there's a way we can confirm this is working? Do we already have tests covering that?